ProteoformSuiteInternal/DeltaMassPeak.cs

```diff
@@ -20,21 +19,28 @@ public List<ProteoformRelation> grouped_relations
                 this.peak_deltaM_average = value.Select(r => r.delta_mass).Average();
                 if (grouped_relations[0].connected_proteoforms[1] is TheoreticalProteoform)
                 {
-                    this.peak_accepted = this.peak_relation_group_count >= Lollipop.min_signal_noise * Lollipop.et_average_noise_level;
-                    this.peak_group_fdr = Lollipop.et_average_noise_level / (Lollipop.et_average_noise_level + this.peak_relation_group_count) * 100;
+                    this.peak_accepted = this.peak_relation_group_count >= Lollipop.min_peak_count_et;
                 }
                 else
                 {
-                    this.peak_accepted = this.peak_relation_group_count >= Lollipop.min_signal_noise * Lollipop.ee_average_noise_level; 
-                    this.peak_group_fdr = Lollipop.ee_average_noise_level / (Lollipop.ee_average_noise_level + this.peak_relation_group_count) * 100;
+                    this.peak_accepted = this.peak_relation_group_count >= Lollipop.min_peak_count_ee;
                 }
             }
         }
         public double peak_deltaM_average { get; set; }
         public int peak_relation_group_count { get; set; }
         //public double decoy_relation_count { get; set; }
-        public double peak_group_fdr { get; set; }
-        public bool peak_accepted { get; set; }
+        public double peak_group_fdr { get; set; }  //TODO: determine how to set FDR 
```

Comment:
I still think we should allow people to use the previously published method of determining the FDR (ED and EF) if they don't mind the additional computational time.